src/Microsoft.Win32.Registry.AccessControl/pkg/Microsoft.Win32.Registry.AccessControl.pkgproj

```diff
@@ -1,15 +1,16 @@
 ﻿<Project DefaultTargets="Build">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
   <ItemGroup>
     <ProjectReference Include="..\ref\Microsoft.Win32.Registry.AccessControl.csproj">
-      <SupportedFramework>net461;netcoreapp2.0;$(AllXamarinFrameworks)</SupportedFramework>
+      <SupportedFramework>netcoreapp2.0;$(AllXamarinFrameworks)</SupportedFramework>
     </ProjectReference>
     <ProjectReference Include="..\src\Microsoft.Win32.Registry.AccessControl.csproj" />
     <HarvestIncludePaths Include="ref/net46;lib/net46;runtimes/win/lib/net46" />
+    <HarvestIncludePaths Include="ref/net461;lib/net461;runtimes/win/lib/net461" />
```

Comment:
So what happens when I build against the latest netstandard2.0 reference which builds live but run on net461?

Reply:
Because of the ProjectExclusions in the Directory.Build.props the Microsoft.Win32.Registry and Microsoft.Win32.Registry.AccessControl are not build at all during the .NET Framework vertical build and instead these assemblies are binplaced into the testhost/runtime folder from a harvested package. We did the same for System.Memory. This does not affect sourcebuild.

Reply:
I'm talking about a customer consuming this package.  I believe they will get a netstandard ref versioned higher than the net461 facade, and thus a ref-def mismatch.

Reply:
Looks like I'm missing some context here. I thought if a target framework specific ref assembly exists in the package it will always be picked over a netstandard one, independent of assembly version differences.

Reply:
Exactly.  No imagine someone builds a netstandard2.0 library, say MyLibrary.dll, that references this package.  Now imagine they use that MyLibrary.dll in a net461 project.   This is completely supported and in fact the whole purpose of netstandard.  MyLibrary.dll will have a reference to a higher version than is supplied by the package (at both compile and runtime) and thus they'll hit an error.  If you need more context, simply test the scenario I'm suggesting.

Reply:
So please try this out so you understand what I'm talking about.  I think the implications are that we need to harvest more (like the netstandard2.0 ref and lib), or not do this.

Reply:
Did you have a chance to try this?

Reply:
unfortunately not. I would have wanted to get this in but there's too much going on right now. I don't think I can finish this before tomorrow.